src/components/Table/Table.stories.tsx

```diff
@@ -235,53 +201,323 @@ export const All: Story = () => (
               <Td>cell</Td>
-            </tr>
-            <tr>
               <Td>cell</Td>
               <Td>cell</Td>
               <Td>cell</Td>
             </tr>
             <tr>
+              <Td rowSpan={2}>rowSpan=2</Td>
+              <Td>cell</Td>
+              <Td>cell</Td>
+              <Td>cell</Td>
               <Td>cell</Td>
               <Td>cell</Td>
-              <Td>
-                multi
-                <br />
-                line
-              </Td>
             </tr>
             <tr>
-              <Td nullable={true}></Td>
-              <Td nullable={true}>not null</Td>
-              <Td nullable={false}></Td>
+              <Td>cell</Td>
+              <Td>cell</Td>
+              <Td>cell</Td>
+              <Td>cell</Td>
+              <Td>cell</Td>
             </tr>
           </tbody>
         </Table>
-      </Base>
-    </li>
-
-    <li>
-      Table with empty state
-      <Base>
-        <Table>
-          <thead>
-            <tr>
-              <Th>cell</Th>
-              <Th>cell</Th>
-              <Th>cell</Th>
-            </tr>
-          </thead>
-          <EmptyTableBody>
-            <Text>お探しの条件に該当する項目はありません。</Text>
-            <Text>別の条件をお試しください。</Text>
-          </EmptyTableBody>
-        </Table>
-      </Base>
-    </li>
-  </Ul>
-)
+      </li>
+      <li>
+        Table on Base
+        <Base>
+          <Table>
+            <thead>
+              <tr>
+                <Th>cell</Th>
+                <Th>cell</Th>
+                <Th>cell</Th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <Td>cell</Td>
+                <Td>cell</Td>
+                <Td>cell</Td>
+              </tr>
+              <tr>
+                <Td>cell</Td>
+                <Td>cell</Td>
+                <Td>cell</Td>
+              </tr>
+              <tr>
+                <Td>cell</Td>
+                <Td>cell</Td>
+                <Td>
+                  multi
+                  <br />
+                  line
+                </Td>
+              </tr>
+              <tr>
+                <Td nullable={true}></Td>
+                <Td nullable={true}>not null</Td>
+                <Td nullable={false}></Td>
+              </tr>
+            </tbody>
+          </Table>
+        </Base>
+      </li>
+      <li>
+        Table with empty state
+        <Base>
+          <Table>
+            <thead>
+              <tr>
+                <Th>cell</Th>
+                <Th>cell</Th>
+                <Th>cell</Th>
+              </tr>
+            </thead>
+            <EmptyTableBody>
+              <Text>お探しの条件に該当する項目はありません。</Text>
+              <Text>別の条件をお試しください。</Text>
+            </EmptyTableBody>
+          </Table>
+        </Base>
+      </li>
+    </Ul>
+  )
+}
 All.storyName = 'all'
 
+export const WithReel: Story = () => {
+  return (
+    <Ul>
+      <li>
+        TableReel
+        <TableReel style={{ whiteSpace: 'nowrap' }}>
```

Comment:
機能に影響するわけではなさそうなので取ったほうがよさそう

```suggestion
        <TableReel>
```

Reply:
Table を nowrap にすることがそれなりに多いので、Table に props で指定できるといいんですかね（別チケで!

Reply:
のほうがよさそうですねー